transformer/transformer.go

```diff
@@ -220,43 +220,67 @@ func buildDescription(doc esResponseHit) description {
 	}
 }
 
-func buildMatches(hl esHighlight) matches {
+func buildMatches(hl esHighlight) *matches {
 	var matches matches
 
-	for _, m := range hl.DescriptionTitle {
-		matchDetails, _ := findMatches(m)
-		matches.Description.Title = append(matches.Description.Title, matchDetails...)
+	if highlights := hl.DescriptionTitle; highlights != nil {
+		titleMatches := make([]matchDetails, 0, len(*highlights))
```

Comment:
Could simplify this to either of the following:

`var titleMatches  []matchDetails`
or
`titleMatches := []matchDetails{}`

It is not necessary to define the length of the array